aidial_adapter_bedrock/llm/model/stabililty/v1.py

```diff
@@ -99,7 +99,7 @@ async def save_to_storage(
     return attachment
```

Comment:
`save_to_storage` could be imported from `aidial_adapter_bedrock.llm.model.stabililty.storage`

Reply:
Fixed